### ISSUE TITLE
Add option for prepending timestamp to generated file

### ISSUE THREAD

**Issue by brookback** [dangcuuson/graphql-schema-typescript#9]
I love this tool! 😍 It was exactly what I looked for. Thanks so much!

It would be cool though with an option (or even as a default) to have a timestamp added to the top of the generated types `.ts` file, in order to quickly diff the version at a glance.

Something like:

```ts
import { GraphQLResolveInfo } from 'graphql';

/**
 * This file is auto-generated by graphql-schema-typescript
 * Please note that any changes in this file may be overwritten
 *
 * Generated at 2018-07-20T23:01:01
 */

/* tslint:disable */ 
/*******************************
 *                             
 *          TYPE DEFS          
 *                             
 *******************************/
```

Let me know if I can provide a PR for this.

**Comment by dangcuuson** [comment]
Hi there,

Thank you for using and loving this small lib :smile: I agree that having the timestamp up there would make it easier to compare file changes.

I think having it in an option would be nice, since some people (like me) run the lib in `prestart` hook so it may be annoying to see the timestamp changes every time (not in my case since I git ignored it, but maybe some others don't).

PRs are always welcomed :+1:

**Comment by brookback** [comment]
Yep, agreed on opt-in. Perhaps an option in `GenerateTypescriptOptions` that can be configured on the command line, such as:

```
graphql-schema-typescript generate-ts --timestamp
```
